Update README.md (#1)
Nola-Ventures/.github: profile/README.md

## Hi there! 👋

> Welcome to the Nola Ventures OÜ organization on GitHub. We are a consulting company specializing in

Reviewer: ```suggestion
Welcome to the Nola Ventures OÜ organization on GitHub. We are a consultancy specializing in
```

> * ☁️ Cloud enviornments

Reviewer: ```suggestion
* ☁️ Cloud development
```

* 💻 Microservice development 

> * 📈 Data analystics

Reviewer: ```suggestion
* 📈 Data analytics
* ⚙️ IoT
```

Reviewer: Vielleicht sollten wir so eine Auflistung auch mal standardisiert erstellen und dann auf linkedin github etc immer gleich verwenden.

#### Contributions 
For any suggestions, please open a new issue with all the relevant details. Once you have created your issue, we will make sure to assign the correct label to it, which will help us to categorize your issue:

* 🐞 use "bug" for something that is not working
* 📖 use "documentation" for any feedback about the documentation
* 💡 use "idea" for every new idea, feature request or suggestion
* ❓ use "support" for questions or anything that needs further assistance

> 

Reviewer: Die ganze Sektion Contributions macht ja keinen Sinn für eine Github Organisation. Das sollte komplett raus.

Author: Done

## Hi there! 👋

> Welcome to the Nola Ventures OÜ organization on GitHub. We are a consultancy specializing in

Reviewer: ```suggestion
Welcome to the Nola Ventures on GitHub. We are a consultancy specializing in
```

* ☁️ Cloud development
  - [Amazon Web Services](https://aws.amazon.com/de/?nc2=h_lg) (AWS)
  - [Cumulocity IoT](https://www.softwareag.com/en_corporate/platform/iot/iot-analytics-platform.html?utm_source=google&utm_medium=cpc&utm_campaign=iot_smart-products&utm_region=hq&utm_subcampaign=stg-2&utm_content=stg-2_webpage_cumulocity-iot-platform&gclid=Cj0KCQiAiJSeBhCCARIsAHnAzT-LQMBKSE51J01TtfuxPXJNVNdLvu0UPh-bx5oK3voYmssw1NsqyzIaAjNAEALw_wcB)
* 💻 Microservice development 
  - Golang & Python
  - Docker containerization
* 📈 Data analytics
  - Spreadsheets
  - PowerBi & Tableau
  - SQL database
  - Python analytics scripts
* 📡 Internet of Things (IoT)

>   - Device registration

Reviewer: ```suggestion
  - Device integration & registration process
```

>   - Frontend application

Reviewer: ```suggestion
  - Frontend development
```

>   - Sensor measurements analysis

Reviewer: ```suggestion
  - Sensor measurement analysis
```
